File: 01_Language/01_Functions/php/ctype_space.php
<?php

$strings = array(
    'string1' => "\n\r\t",
    'string2' => "\narf12",
    'string3' => '\n\r\t', // note the single quotes
    'string4' => ' '
);
foreach ($strings as $name => $testcase) {
    if (ctype_space($testcase)) {
        echo "The string '$name' consists of whitespace characters only.\n";
    } else {
        echo "The string '$name' contains non-whitespace characters.\n";
    }
}

File: 01_Language/01_Functions/python/ctype_space.py
# coding: utf-8

strings = {
    'string1': "\n\r\t",
    'string2': "\narf12",
    'string3': '\n\r\t',  # note the single quotes
    'string4': ' '
}
for k, v in strings.items():
    if v.isspace():
        print("The string " + k + " consists of whitespace characters only.")
    else:
        print("The string " + k + " contains non-whitespace characters.")
